app/controllers/kudos_controller.rb

```diff
@@ -46,6 +47,11 @@ def destroy
     redirect_to kudos_url, notice: 'Kudo was successfully destroyed.'
   end
 
+  def correct_employee
```

Comment:
the method name in my opinion is misleading - I would expect, that it is correcting and employee? Consider renaming it to `check_kudo_giver` or something like that, so it is describes what it is doing.

Reply:
good point, renaming